env/test.mk

```diff
@@ -13,8 +13,9 @@ export SCP_OPTS=
 # where to put binary files
 export TESTDIR=$(realpath .)/pkg.jenkins.io
 export WARDIR=/packages/binary/war${RELEASELINE}
+# Concat MSDIR and RELEASELINE in the msi publishing
 export MSIDIR=/packages/binary/windows${RELEASELINE}
-export OSXDIR=export OSXDIR=${TESTDIR}/osx${RELEASELINE}
+export OSXDIR=/packages/osx${RELEASELINE}
```

Comment:
is OSX still needed here? I thought we didn't ship it anymore

Reply:
We don't ship it anymore and we could problaby remove those variables